demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -238,6 +240,12 @@ fi
 echo
 echo "[INFO] Deploy KServe"
 echo
-oc create -f custom-manifests/opendatahub/kserve-dsc.yaml
+# ODH 1.9 use alpha api so this logic needed but from ODH 1.10, this logic must be deleted.
+#oc create -f custom-manifests/opendatahub/kserve-dsc.yaml
+if [[ ${TARGET_OPERATOR_TYPE} == "rhods" ]]; then
+  oc create -f custom-manifests/opendatahub/kserve-dsc.yaml
+else
+  oc create -f custom-manifests/opendatahub/kserve-dsc-v1alpha1.yaml
+fi
```

Comment:
@Jooho 
Do we have a followup ticket for this? As soon as ODH 1.10 is out this if should be removed

/cc @heyselbi

Reply:
Good point!
I created a ticket to follow up. : https://github.com/opendatahub-io/caikit-tgis-serving/issues/87